### ISSUE TITLE
[Idea] Implement I18n

### ISSUE THREAD

**Issue by casi** [casi/flatshare_app#17]
...initially for the two most important languages: German and English

**Comment by casi** [comment]
@cameronnorman maybe you can help ('cause of my school english). :D

**Comment by cameronnorman** [comment]
Sure! So Initially we would like the application to support two languages:
- English
- German

In order to do this we require to incorporate I18n into the rails application. Rails has good support for this but the views need to be updated to use it.

For any questions regarding how to implement this feature, please ask

**Comment by casi** [comment]
Yeah, I know. But I've looked at that before. It is mostly hard work.
I also don't do it right away. But is definitely a medium-term goal. But thanks in advance!

**Comment by casi** [comment]
Yeah, I know. But I've looked at that before. It is mostly hard work.
I also don't do it right away. But is definitely a medium-term goal. But thanks in advance!

**Comment by casi** [comment]
@cameronnorman I've started a bit before. So we have kinda foundation. -> Commit: [4ce0cce](https://github.com/casi/flatshare_app/commit/4ce0cce84df216d78a55af4af1a08a7d2996ccd5)
If you want to participate, please tell me which views you plan to work with, so that we don't do anything twice.. (If you have time and you feel like, you are welcome to continue.) 😉

**Comment by casi** [comment]
Build the locale switcher. The admin dashboard is still missing, especially the error messages. 
And: All locale test cases are still missing.

**Comment by casi** [comment]
Added I18n with EN & DE locales. -> #40
